### ISSUE TITLE
DRY the releasability workflow.

### ISSUE THREAD

**Issue by krsna-m** [knative/release#193]
https://github.com/knative/release/pull/192 gets us off of knobots and a releasability workflow on each of the repos which is great. However, it has a workflow per repo which can be simplified into a single workflow that reads from a single `repo.yaml` config to check which repos to run against. The down side of this is that  badges https://github.com/knative/release/blob/main/TIMELINE.md wont work because they need a [workflow per badge](https://docs.github.com/en/actions/monitoring-and-troubleshooting-workflows/adding-a-workflow-status-badge) to show. before #192, to check the status without the badges would require one to look at the action on each of the repos which was a daunting task thus the badges were very valuable. Post #192, the releasibility status of all the repos can be seen from the actions pane of the release repo thus making the need for the badges obsolete.

TLDR: #192 has a lot of workflows and a workflow generator which can all be simplified to just one workflow. It will remove the badges, but we don't need the badges because one can just check the status of the releasibility workflow on release repo itself thus making the badges redundant.

**Comment by dprotaso** [comment]
Closing this out due to in-action.

I also think there's still value in seeing the badges
